frontend/src/LoginPage/LoginPage.jsx

```diff
@@ -199,7 +199,7 @@ class LoginPage extends React.Component {
                         onChange={this.handleOnCheck}
                         data-cy="checkbox-input"
                       />
-                      <label className="form-check-label" htmlFor="check-input" data-cy="show-password-label">
+                      <label className="form-check-label" htmlFor="check-Input" data-cy="show-password-label">
```

Comment:
Please change the `htmlFor` value back to "check-input" while the id of checkbox input is "check-input". If you change the `htmlFor` value, we will not be able to `enable or disable` the checkbox by clicking the label. so, please use CSS properties like 
```
.show-password-label {
 width: fit-content;
}
```

Reply:
Hey @shah21 the issue I fixed through this PR is that we should not be able to enable or disable checkout by clicking on label show password #3267